spec/controllers/state_file/questions/az_qualifying_organization_contributions_controller_spec.rb

```diff
@@ -9,6 +9,51 @@
     sign_in intake
   end
 
+  describe ".show?" do
+
+    context "when the intake has charitable contributions" do
+      before do
+        intake.charitable_contributions_yes!
+      end
+
+      context "when the intake has cash contributions" do
+
+        before do
+          intake.update(charitable_cash_amount: 1)
+        end
+
+        it "shows" do
+          expect(described_class).to be_show(intake)
+        end
+
+      end
+
+      context "when the intake does not have charitable contributions" do
```

Comment:
is this supposed to be `does not have cash contributions`?

Reply:
yes, whoops

Reply:
yes, whoops